ci/jenkins/test.sh

```diff
@@ -38,6 +38,7 @@ IP_MODE=""
 K8S_VERSION="1.27.2-00"
 WINDOWS_YAML_SUFFIX="windows"
 WIN_IMAGE_NODE=""
+echo "" > WIN_DHCP
```

Comment:
Just noticed this, why doesn't it just declare WIN_DHCP="" but use echo? I don't mean to change it in a cherry-pick PR but want to understand it. If there is no specific reason, perhaps let's change it in the main branch as this is not a common way to declare a variable.